core/src/main/java/com/segment/analytics/kotlin/core/Analytics.kt

```diff
@@ -405,6 +406,12 @@ open class Analytics protected constructor(
      */
     fun <T: Plugin> find(plugin: KClass<T>): T? = this.timeline.find(plugin)
 
+    /**
+     * Retrieve the first match of registered destination plugin by name. It finds
+     * @param destination [String]
+     */
+    fun find(destination: String): DestinationPlugin? = this.timeline.find(destination)
```

Comment:
let's change param name from destination to `destinationName` / `destinationKey`